packages/react-next/src/components/Modal/Modal.base.tsx

```diff
@@ -235,31 +344,70 @@ export class ModalBase extends React.Component<IModalProps, IDialogState> implem
             )}
           </div>
         )}
-        <div ref={this._allowScrollOnModal} className={classNames.scrollableContent} data-is-scrollable={true}>
-          {dragOptions && this.state.isModalMenuOpen && (
+        <div ref={allowScrollOnModal} className={classNames.scrollableContent} data-is-scrollable>
+          {dragOptions && isModalMenuOpen && (
             <dragOptions.menu
               items={[
```

Comment:
This may not be a new issue but these items ought to be memoized (and you can move the callback definitions inside the memoization, so they don't need to be memoized separately)